airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -18,7 +18,7 @@
 
 
 class IntercomStream(HttpStream, ABC):
-    url_base = "https://api.intercom.io/"
+    url_base = "https://api.intercom.io"
```

Comment:
this is concatenated with stream paths below to form `https://api.intercom.io/<path>` -- is it safe to do this?

Reply:
Doesn't matter `urljoin` will parse correctly with `/` or without. 

The problem happens in check connection operation where is concatenating using f-string and not urljoin.
https://github.com/airbytehq/airbyte/blob/266a491c8e3236b7e6603f0c34aa7ac1abf0a4c3/airbyte-integrations/connectors/source-intercom/source_intercom/source.py#L416-L417

Reply:
@sherifnada @marcosmarxm Would you guys prefer it if I change it to a `urljoin`?

Reply:
That would be better imo

Reply:
Updated!